ajax/libs/require-cs/package.json

```diff
@@ -23,5 +23,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/requirejs/require-cs"
-  }
+  },
+  "license": "CC0 v1.0"
```

Comment:
This should be `MIT` according to https://github.com/requirejs/require-cs#license